src/main/_docs/admin-guide/admin-runbook.md

```diff
@@ -32,22 +31,205 @@ Zookeeper is a key-value store that is needed by Swarm in a clustered Codenvy se
 -A INPUT -m state --state NEW -m udp -p udp --dport 4789 -j ACCEPT
 ```
 
-**Step 2**: Create directories for Zookeeper:
+**Step 2**: Change the Docker daemon configuration (subsituting the hostname for your instance and the appopriate network adapter):
 
 ```
-mkdir -p var/lib/zookeeper/datalog
+DOCKER_NETWORK_OPTIONS=' --bip=172.17.42.1/16 -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock --cluster-store=zk:// codenvy.com:2181 --cluster-advertise=eth0:2375'
 ```
 
-**Step 3**: Start the zookeeper service:
+# Network Infrastructure
+We have 2 classes of instances:
+
+1. Master Nodes used for running internal Codenvy services.
+2. Workspace Nodes that runs user workspaces.
+
+We suggest separating traffic bewteen two subnets: one for Master and one for Workspace Nodes. This keeps the traffic purpose and security properly segregated and allows the use of subnet masks, rather than individual IPs when writing firewall rules. VNET will use addresses `10.0.0.0/8` and two subnets `10.1.0.0/16` for Master and 10.2.0.0/16 for Workspace Nodes.
+
+# Storage
+Cloud-based installs (AWS, Google Cloud, etc...) can scale disk space. We suggest LVM and XFS with caching for read and write.
+
+Workspaces Nodes require fast I/O access to give developers the best experience. This is especially important with interpreted languages like node.js and PHP that require access to a large number of small files. We suggest using SSD for all nodes. 
+
+For Workspace Nodes local storage is preferred as it will provide the best performance.
+
+For the Master Node, LVM or RAID storage is recommended for redundancy. NAS can also be used. If you are using AWS, we recommend LVM with snapshotting turned on for faster backups of the key data.
+
+## Directories
+
+For Master Node:
+
+```
+/var/lib/pgsql
+/var/lib/docker
+/var/lib/docker-distribution
+/home/codenvy/codenvy-data/fs
+/home/codenvy/codenvy/instance/data/codenvy/che-machines
```

Comment:
These /home/codnevy directories do not exist in Dockerized.  This stuff is mounted to :/data now.